widgets-samples/cc/samples-cc-react-app/src/App.tsx

```diff
@@ -556,6 +563,16 @@ function App() {
                         setintegrationEnv(!integrationEnv);
                       }}
                     />
+                    <Checkbox
+                      checked={multiPartyConferenceEnabled}
+                      aria-label="multi party conference enabled checkbox"
+                      id="multi-party-conference-enabled-checkbox"
+                      label="Enable Multi-Party Conference"
```

Comment:
Again, should we make this a conference toggle?
```suggestion
                      label="Enable Conference Feature"
```

Reply:
Addressed